pages/docs/modules/exec.mdx

```diff
@@ -0,0 +1,355 @@
+# exec
+
+## Functions
+
+### command
+
+```go filename="Function signature"
+command(file string, args ...string) exec.command
+```
+
+Creates a new `exec.command` targeting a specific `file` to execute,
+followed by arguments to execute the command with.
+
+Nothing is executed by this function alone, but instead only a [`exec.command`](#type-command)
+object is returned. To execute `file`, you must call the [`run`](#commandrun)
+method on the [`exec.command`](#type-command) object.
+
+The lookup if `file` executable exists is also not performed until the [`run`](#commandrun)
```

Comment:
I found out that Nextra uses https://github.com/Flet/github-slugger for its header slugs, which trims away the dot.

If slug-styling matters to you (i.e would be better with `[...](#command.run)` or `[...](#command-run)`) then we need to change the header texts. Such as using space instead of dot to get the `#command-run` variant